### ISSUE TITLE
[BUG] Error: Invalid argument

### ISSUE THREAD

**Issue by romulasry** [lutris/lutris#1130]
Running a game in DXVK stops at a bug:

Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/lutris/thread.py", line 355, in watch_children
    resume_stop = self.stop()
  File "/usr/lib/python3.6/site-packages/lutris/thread.py", line 242, in stop
    self.restore_environment()
  File "/usr/lib/python3.6/site-packages/lutris/thread.py", line 225, in restore_environment
    del os.environ[key]
  File "/usr/lib64/python3.6/os.py", line 680, in __delitem__
    self.unsetenv(encodedkey)
OSError: [Errno 22] Invalid argument

**Comment by Danie10** [comment]
I'm getting the identical error during a DXVK install for Assetto Corsa while installing. Is there any solution or workaround for it?

OSError(22, 'Invalid argument')

**Comment by strycore** [comment]
No, not with this little information given to us.

**Comment by Danie10** [comment]
Yes that's my challenge too. Not sure where to get the log as Lutris gives an option to view logs but its a blank window. I thought Lutris would install a clean wine bottle etc for each game. I'm trying now to follow a guide to setting up Lutris and I see it says I should installed DXVK-bin, which another person had told me was not necessary as Lutris should have taken care of that.

So if you can tell me where to find a log file, I can get more detail behind this error.

**Comment by strycore** [comment]
I didn't realize this bug was closed by the original reporter and not by a fix.
You do not need DXVK installed prior to using it with Lutris, the client downloads the most recent version and anything else installed on your system will be ignored.
Logs are available in ~/.cache/lutris, although you might get the same error as the initial report.
Note that while I can fix the crash itself, it's unlikely that this will fix the underlying issue.

**Comment by Danie10** [comment]
@strycore thanks the good news is I got it to work 100% now! Noted what you said about the DXVK but this is what I did in the end (most lilley not the DXVK then):
1. Installed DXVK under Linux.
2. Deleted previous AC completely from Steam as well as in Lutris library.
3. Noticed my Nvidia drivers were two branches behind and ran the upgrade for that, and rebooted.
4. Used the DXVK install script from Lutris... ran a long time saw it install Steam for Win, Wine, etc with quite a few errors (like running out of storage space when I had 750GB free) but after it fully finished, game started first ttime and everything including my wheel is working (OK not FFB but I'm waiting to get Logitech G29 which I hope will then work with FFB).

So I'm really not sure what caused that previous install error right at the beginning of the script. May even have been the older Nvidia drivers. But I'm really happy now.
